### ISSUE TITLE
Add readthedocs badge to extension readme

### ISSUE THREAD

**Issue by adswa** [adswa/git-annex-remote-osf#17]
This is the code that needs to be added to the top of the README:

```
[![Documentation Status](https://readthedocs.org/projects/datalad-osf/badge/?version=latest)](http://docs.datalad.org/projects/osf/en/latest/?badge=latest)
```

**Comment by mih** [comment]
Maybe also add something like this:

```diff
diff --git a/README.md b/README.md
index f577ca5..32432d2 100644
--- a/README.md
+++ b/README.md
[![Travis tests status](https://secure.travis-ci.org/datalad/datalad-osf.png?branch=master)]

-EXPLAIN ME
+This software is a [DataLad](http://datalad.org) extension that equips DataLad
+with tools to interoperate with the [Open Science Framework
+(OSF)](https://osf.io)
+(see [documentation](http://docs.datalad.org/projects/osf) for more information).
```

**Comment by adswa** [comment]
Thanks so much @TheDragon246!
Would you please:
- close this issue , now that it is fixed (hints: please assign yourself the next time, if you say "fixes #17" instead of "resolve issue #17" and do a PR, this would close the issue when merged :slightly_smiling_face:)
- could you also add this badge to https://github.com/datalad/datalad-osf? :) Apart from the badge, it would be good to include the text @mih posted in the comment above into the README. If you are up to it, then ... 
    - first create a new issue in  https://github.com/datalad/datalad-osf (something like: missing documentation info and badge), 
    - and try providing the change as a PR from a new branch, and
    - include the "fixes" keyword either in the commit message, or in the PR description to automatically close the issue once your PR is merged.

Let me know if that's okay, and also if you run into trouble! :)

hey @all-contributors please add @TheDragon246 for doc maintenance

**Comment by adswa** [comment]
hey @all-contributors please add @TheDragon246 for doc and maintenance

**Comment by adswa** [comment]
hey @TheDragon246, would you still be up to doing this?

**Comment by TheDragon246** [comment]
I forgot to fork this github and subsequently couldn't track. Will definitely do it. :)

**Comment by adswa** [comment]
Awesome, thank you so much @TheDragon246 ! :)

**Comment by TheDragon246** [comment]
As I can see, the main readme.md has already been updated by Michael. Should i just close this issue then?

**Comment by adswa** [comment]
Ah, then yes, thanks.

**Comment by TheDragon246** [comment]
@adswa Added a PR to close this. https://github.com/adswa/git-annex-remote-osf/pull/20
Please suggest if I did something errorneous or there is a better way to do it. I don't think I could assign the PR to myself as I don't have write access.
Thanks 🙂
